.github/workflows/ci.yml

```diff
@@ -326,6 +326,7 @@ jobs:
           npm install --ignore-scripts
           npx napi build --release --target x86_64-unknown-linux-musl --platform --zig
           mkdir -p /tmp/build
+          ls -l
```

Comment:
should we add this or was it for debugging purposes?

Reply:
right :" )sorry

Reply:
```suggestion
```